lib/tasks/wikirate/test.rake

```diff
@@ -1,17 +1,21 @@
 require "colorize"
+require File.expand_path "../importer", __FILE__
 
-# require 'pry'
 namespace :wikirate do
   namespace :test do
-    db_path = File.join Wagn.root, "test", "seed.db"
-    testdb = ENV["DATABASE_NAME_TEST"] ||
-             ((t = Wagn.config.database_configuration["test"]) &&
-             t["database"])
+    full_dump_path = File.join Wagn.root, "test", "seed.db"
+    base_dump_path = File.join Wagn.root, "test", "base_seed.db"
```

Comment:
base_dump is incomplete test db (with imports but w/o migrations and test data). If something breaks at the end of the seed generation you can start with that instead of importing everything again.